lib/models/api-key.ts

```diff
@@ -115,7 +115,9 @@ const getApiKeysModel = function(
 	 */
 	exports.getAll = function(
 		options: BalenaSdk.PineOptionsFor<BalenaSdk.ApiKey> = {},
-		callback: (error?: Error, apiKeys?: BalenaSdk.ApiKey[]) => void,
+		callback:
```

Comment:
I think this should be
```suggestion
		callback?: (error?: Error, apiKeys?: BalenaSdk.ApiKey[]) => voi
```
so that can omit it as well as explicitly specifying `undefined`